core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -509,19 +509,13 @@ private void updateAuthorizationFailures(Map<KeyExtent,SecurityErrorCode> author
 
       synchronized (this) {
         somethingFailed = true;
-        mergeAuthorizationFailures(this.authorizationFailures, authorizationFailures);
```

Comment:
Would a comment be appropriate?  The method name explains what is happening, when merged it looks harder to reason why this is being done.

Reply:
Sure, I can add that.